Client/mods/deathmatch/logic/CClientVehicle.h

```diff
@@ -145,6 +145,12 @@ struct SVehicleComponentData
     bool    m_bVisible;
 };
 
+static const std::map<eClientVehicleType, std::string> g_vehicleTypePrefixes{
```

Comment:
std::map is excessive here. There is 10 enum entries that can be fully covered by a std::array.